<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLPinholeCameraIntrinsicsWidget</class>
 <widget class="QWidget" name="qMRMLPinholeCameraIntrinsicsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>351</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ctkCollapsibleButton" name="collapsibleButton_Camera">
     <property name="text">
      <string>Camera</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="qMRMLNodeComboBox" name="comboBox_CameraSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLPinholeCameraNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkCollapsibleButton" name="collapsibleButton_Details">
        <property name="text">
         <string>Details</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_IntrinMatrix">
           <property name="text">
            <string>Intrinsics Matrix:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="container_Intrinsics" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="ctkMatrixWidget" name="MatrixWidget_CameraMatrix">
              <property name="columnCount">
               <number>3</number>
              </property>
              <property name="rowCount">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>-1000000000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1" rowspan="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_DistCoeffs">
           <property name="text">
            <string>Distortion Coefficients:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QWidget" name="container_DistCoeffs" native="true">
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="ctkMatrixWidget" name="MatrixWidget_DistCoeffs">
              <property name="columnCount">
               <number>5</number>
              </property>
              <property name="rowCount">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>-1000000000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_Tracker">
           <property name="text">
            <string>MarkerToImageSensor:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QWidget" name="container_MarkerToImage" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="1" rowspan="2">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="ctkMatrixWidget" name="MatrixWidget_MarkerToImageSensor">
              <property name="minimum">
               <double>-1000000000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_CameraPlaneOffset">
           <property name="text">
            <string>Camera Plane Offset (mm):</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QWidget" name="container_CameraPlaneOffset" native="true">
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="ctkMatrixWidget" name="MatrixWidget_CameraPlaneOffset">
              <property name="columnCount">
               <number>3</number>
              </property>
              <property name="rowCount">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>-1000000000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1" rowspan="2">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkMatrixWidget</class>
   <extends>QWidget</extends>
   <header>ctkMatrixWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_DistCoeffs"/>
 </buttongroups>
</ui>
